src/modules/visits/types.ts

```diff
@@ -87,3 +87,9 @@ export interface Note {
 	note: string;
 	createdAt: string;
 }
+
+export interface AccessStatus {
+	spaceId: string;
+	visitorId: string;
+	status: VisitStatus;
+}
```

Comment:
Wouldn't we just return the status, since the user known their own id and they have to provide the space id in the url.
So on the route i would return:
```
{
	status: VisitStatus;
}
```

on the service i would return:
```
VisitStatus
```